src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java

```diff
@@ -285,6 +314,33 @@ public SELF containsOnly(double... values) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual array contains only the values of the given array and nothing else, in any order.
+   * <p>
+   * Example:
+   * <pre><code class='java'> // assertions will pass
+   * assertThat(new double[] { 1.0 , 2.0 }).containsOnly(new Double[] { 1.0, 2.0 });
+   * assertThat(new double[] { 2.0, 1.0 }).containsOnly(new Double[] { 1.0, 2.0 });
+   * assertThat(new double[] { 1.0, 1.0, 2.0 }).containsOnly(new Double[] { 1.0, 2.0 });
+   *
+   * // assertions will fail
+   * assertThat(new double[] { 1.0, 2.0 }).containsOnly(new Double[] { 2.0 });
+   * assertThat(new double[] { 1.0 }).containsOnly(new Double[] { 1.0, 2.0 });</code></pre>
+   *
+   * @param values the given values.
+   * @return {@code this} assertion object.
+   * @throws NullPointerException if the given argument is {@code null}.
+   * @throws IllegalArgumentException if the given argument is an empty array.
+   * @throws AssertionError if the actual array is {@code null}.
+   * @throws AssertionError if the actual array does not contain the given values, i.e. the actual array contains some
+   *           or none of the given values, or the actual array contains more values than the given ones.
+   */
+  public SELF containsOnly(Double[] values) {
```

Comment:
Please also add
```java
public SELF containsOnly(Double[] values, Offset<Double> precision)
```

Reply:
https://github.com/assertj/assertj-core/blob/c0f48d3858e047ed0421a36c435e6f1b5503edcb/src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java#L312

It's already there